presto-main/src/main/java/com/facebook/presto/sql/planner/PlanFragmenter.java

```diff
@@ -1084,26 +1085,32 @@ private GroupedExecutionProperties processChildren(PlanNode node)
             //     * if all children is "recovery eligible", this node is "recovery eligible"
             //     * if any child is "not recovery eligible", this node is "not recovery eligible"
             boolean anyUseful = false;
-            OptionalInt totalLifespans = OptionalInt.empty();
+            OptionalInt totalLifespansOptional = OptionalInt.empty();
```

Comment:
wow, can we just keep `totalLifespans` and use `totalLifespans.getAsInt()` every time the value is used ?